how-to/how-to-rename-field-in-content-type.md

```diff
@@ -0,0 +1,53 @@
+---
+title: How to rename a field in content type
+description: Rename a field in content type using mongodb '$rename' operator
+type: how-to
+page_title: Rename a field in a content type
+slug: rename-field-in-content-type
+position:
+tags: platform-core, rename-field, backendservices, content-type

```

Comment:
Please use only the available tags in the tags.md file or add the new ones there. Really, we do not need the new tags, imo, other than content-type